### PR TITLE
hidden edit and delete buttons

### PR DESCRIPTION
**Pull request by brac193**

### CODE REVIEW

File: app/controllers/bars_controller.rb [hunk 1]
```diff
@@ -35,16 +36,23 @@ def update
   end
 
   def destroy
-    @bar.destroy
+    @bar = Bar.find(params[:id])
```
Comment (wbucko) [inline app/controllers/bars_controller.rb]: You sure that destroy method should only find proper Bar? :)

File: app/controllers/bars_controller.rb [hunk 2]
```diff
@@ -35,16 +36,23 @@ def update
...
+  end
+
+  def bar_params
+    params.require(:bar).permit(:name, :address, :description, :rating, :user_id)
+  end
+
+  def author?
+    @bar = Bar.find(params[:id])
+    redirect_to bars_path if current_user != @bar.user
+  end
```
Comment (wbucko) [inline app/controllers/bars_controller.rb]: Not sure of that belongs to controller.  Consider using Pundit.

File: app/controllers/bars_controller.rb (hunk 1 above)
Comment (brac193) [reply app/controllers/bars_controller.rb]: Fixed :)

File: app/controllers/bars_controller.rb (hunk 2 above)
Comment (brac193) [reply app/controllers/bars_controller.rb]: Well I'm not sure either, if I have enough time will try to use Pundit in separate PR, ok?

**Review by wbucko — APPROVED** [review]

**Pull request merged**